aws/resource_aws_appsync_graphql_api.go

```diff
@@ -159,6 +165,11 @@ func resourceAwsAppsyncGraphqlApiCreate(d *schema.ResourceData, meta interface{}
 		return err
 	}
 
+	err = resourceAwsAppsyncSchemaPut(*resp.GraphqlApi.ApiId, d, meta)
+	if err != nil {
+		return err
```

Comment:
We should return error context here for operators and code maintainers:

```suggestion
		return fmt.Errorf("error creating AppSync GraphQL API (%s) Schema: %s", d.Id(), err)
```